config/initializers/amazon_ses.rb

```diff
@@ -0,0 +1,7 @@
+
+if Rails.env.production?
+  ActionMailer::Base.add_delivery_method(:ses, AWS::SES::Base,
+    access_key_id:      ENV['AMAZON_ACCESS_KEY'],
+    secret_access_key:  ENV['AMAZON_SECRET_KEY']
```

Comment:
What should happen is that the app refuses to start and produces a helpful error message detailing all missing configuration.

But nothing else in the app does that or provides defaults.

We should revisit later and flag as tech debt.